### PR TITLE
Update ruby path to use right ruby distribution

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

File: install.bat [hunk 1]
```diff
@@ -36,7 +36,7 @@ ECHO @ECHO OFF > %ChocolateyInstall%\bin\xslt-config.bat
 
 SET XSLT_INCLUDE=%ChocolateyInstall%\lib\xsltproc\dist\include
 SET XSLT_LIB_DIR=%ChocolateyInstall%\lib\xsltproc\dist\lib
-SET RUBY_BIN=c:\tools\ruby25\bin
+SET RUBY_BIN=c:\tools\ruby23\bin
```
Comment (CAMOBAP) [inline install.bat]: I'm not happy about this solution, I looking for some `choco` command which can get executable path from package name, in progress

**Comment by ronaldtse** [comment]
@CAMOBAP795 is this to be merged or modified?

**Comment by CAMOBAP** [comment]
@ronaldtse it's ready for merge

**Comment by ronaldtse** [comment]
@CAMOBAP795 is there a way to find the currently installed Ruby path without hardcoding it? This means if we upgrade the Ruby package we need to update all the paths again.

**Comment by CAMOBAP** [comment]
> @CAMOBAP795 is there a way to find the currently installed Ruby path without hardcoding it?

There is no solution out of box in chocolatey, we can only get `tools` directory from `%ChocolateyToolsLocation%`

We can implement our own, but how it should work in case if we have many ruby distribution? Should we care about rvm?

**Comment by ronaldtse** [comment]
If RVM is installed, what can we do? Do we just detect if RUBY_BIN is set, and if it is, then we skip the `SET`?

**Comment by CAMOBAP** [comment]
Can be closed because of https://github.com/riboseinc/metanorma-windows-setup/pull/14

**Pull request closed without merging**